server/endpoint/creator/endpoint.go

```diff
@@ -0,0 +1,134 @@
+package creator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	kitendpoint "github.com/go-kit/kit/endpoint"
+	kithttp "github.com/go-kit/kit/transport/http"
+	"github.com/gorilla/mux"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/credentiald/server/middleware"
+	"github.com/giantswarm/credentiald/service/creator"
```

Comment:
We have this specified differently. Please see https://github.com/giantswarm/fmt/blob/master/go/general_style_guide.md#imports.

Reply:
Aligned.